src/components/BottomNavigation.tsx

```diff
@@ -601,406 +575,388 @@ class BottomNavigation extends React.Component<Props, State> {
+
+  const { routes } = navigationState;
+  const { colors, dark: isDarkTheme, mode } = theme;
+
+  const shifting = isShifting();
+
+  const { backgroundColor: customBackground, elevation = 4 }: ViewStyle =
+    StyleSheet.flatten(barStyle) || {};
+
+  const approxBackgroundColor = customBackground
+    ? customBackground
+    : isDarkTheme && mode === 'adaptive'
+    ? overlay(elevation, colors.surface)
+    : colors.primary;
+
+  const backgroundColor = shifting
+    ? index.interpolate({
+        inputRange: routes.map((_, i) => i),
+        //@ts-ignore
+        outputRange: routes.map(
+          (route) => getColor({ route }) || approxBackgroundColor
+        ),
+      })
+    : approxBackgroundColor;
+
+  const isDark = !color(approxBackgroundColor).isLight();
+
+  const textColor = isDark ? white : black;
+  const activeTintColor =
+    typeof activeColor !== 'undefined' ? activeColor : textColor;
+  const inactiveTintColor =
+    typeof inactiveColor !== 'undefined'
+      ? inactiveColor
+      : color(textColor).alpha(0.5).rgb().string();
+
+  const touchColor = color(activeColor || activeTintColor)
+    .alpha(0.12)
+    .rgb()
+    .string();
+
+  const maxTabWidth = routes.length > 3 ? MIN_TAB_WIDTH : MAX_TAB_WIDTH;
+  const maxTabBarWidth = maxTabWidth * routes.length;
+
+  const tabBarWidth = Math.min(layout.width, maxTabBarWidth);
+  const tabWidth = tabBarWidth / routes.length;
+
+  const rippleSize = layout.width / 4;
+
+  return (
+    <View style={[styles.container, style]}>
+      <View style={[styles.content, { backgroundColor: colors.background }]}>
+        {routes.map((route, index) => {
+          if (!loaded.includes(index)) {
+            // Don't render a screen if we've never navigated to it
+            return null;
+          }
+          const focused = navigationState.index === index;
+
+          const opacity = sceneAnimationEnabled ? tabs[index] : focused ? 1 : 0;
+
+          const top = offsets[index].interpolate({
+            inputRange: [0, 1],
+            outputRange: [0, FAR_FAR_AWAY],
+          });
+
+          return (
+            <Animated.View
+              key={route.key}
+              pointerEvents={focused ? 'auto' : 'none'}
+              accessibilityElementsHidden={!focused}
+              importantForAccessibility={
+                focused ? 'auto' : 'no-hide-descendants'
+              }
+              style={[StyleSheet.absoluteFill, { opacity }]}
+              collapsable={false}
+              removeClippedSubviews={
+                // On iOS, set removeClippedSubviews to true only when not focused
+                // This is an workaround for a bug where the clipped view never re-appears
+                Platform.OS === 'ios' ? navigationState.index !== index : true
+              }
+            >
+              <Animated.View style={[styles.content, { top }]}>
+                {renderScene({
+                  route,
+                  jumpTo: jumpTo,
+                })}
```

Comment:
How about shorting it a little?
```suggestion
                {renderScene({ route, jumpTo })}
```